_posts/2016-07-04-imam-week1.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title:  "CrowdCensus Team's Data Exploration"
+date:   2016-07-04
+author: Imam and Myeong
+---
+
+The goal of CrowdSensing Census project in DSSG 2016 is to develop a reliable and general model that can complement census data by making use of other real-time data such as OpenStreetMap data or Cellphone Detail Records (CDR) to determine the social economic conditions in a particular region. This work is meaningful for allowing data freshness and cost efficiency compared to census data, since government-initiated socio-economic statistics are expensive and often outdated (10-year term). 
```

Comment:
"The goal of Crowdsensing..." => "The goal of *the* CrowdSensing..."

Link to OpenStreetMap? 

"This work is meaningful for allowing..." => "This work is meaningful because it improves data freshness..."

I would also cut the last sentence into two sentences, to improve readability.